test/cli/package-multiple-files-behavior/test.sh

```diff
@@ -0,0 +1,2 @@
+# Ensures that Sorbet doesn't complain about the package having behavior defined in multiple files in Stripe mode.
```

Comment:
forgot to replace the "stripe mode" here

Reply:
that'll be our little easter egg